crates/common/build.rs

```diff
@@ -1,7 +1,8 @@
 use std::{env, error::Error};
 
 use chrono::DateTime;
-use vergen::EmitBuilder;
+use vergen::{BuildBuilder, CargoBuilder, Emitter};
```

Comment:
i'd rather keep vergen 8 until 10 is released, because 9 has a lot more dependencies for no gain

Reply:
Reverted to 8: https://github.com/foundry-rs/foundry/pull/11123/commits/81d7e2cc6b67509055f883b953154dea42aeccc1